### DISCUSSION TITLE
Restart is needed after installing packages

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#90] [Concerns/constructive criticism]
Minor hindrance.

**Comment by ghost** [comment]
An even more concerning issue is that editable installs have to be reinstalled and Nodezator restarted so that changes to the code apply.

**Comment by ghost** [comment]
Also, editable installed dependencies need a restart after a change also.

**Comment by KennedyRichard** [comment]
Since you already opened an issue related to this discussion, I'll be closing this discussion, so that we can keep discussing and solve this issue from there instead.
